src/index.html

```diff
@@ -6,7 +6,434 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <img
+      src="images/logo.png"
+      alt="header icon"
+      class="header__icon"
+      >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">ГАДЖЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">ПЛАНШЕТЫ</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">ФОТО</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main-container main-container--page">
+      <div class="card card--page" data-qa="card">
```

Comment:
Check BEM rules. Should be:
<img width="346" alt="image" src="https://user-images.githubusercontent.com/52370890/169231580-08dc650e-74b4-4fb8-aee6-fa41b241ae82.png">